Plots/Contours/NCL_conwomap_1.py

```diff
@@ -0,0 +1,63 @@
+"""
+NCL_conwomap_1.py
+=================
+This script illustrates the following concepts:
+   - Drawing a simple line contour plot
+   - Drawing a line contour plot with multiple colors
+   - Increasing the thickness of contour lines
```

Comment:
This concept is from NCL script but it seems that no thickness adjustment is done to the contour plot in here. Playing with the argument `linewidths` in `u.plot.contour()` at Line 46 would be good to satisfy this concept.

Reply:
Good catch, made use of `linewidths` in the call to `contour`.